src/entries/popup/pages/home/NFTs/NFTDetails.tsx

```diff
@@ -626,8 +627,9 @@ const NFTAccordionAboutSection = ({
   nft?: UniqueAsset | null;
   showFloorPriceExplainerSheet: () => void;
 }) => {
-  const network =
-    nft?.network === 'mainnet' ? 'Ethereum' : capitalize(nft?.network);
+  const networkDisplay = nft?.network
+    ? ChainNameDisplay[chainIdFromChainName(nft?.network)]
+    : '';
```

Comment:
is this `''` just used as a "falsy" value?

Reply:
Yes, just not showing that row if nft is undefined. Just being safe, that shouldn't be the case unless something broke.